pyproject.toml

```diff
@@ -59,6 +59,7 @@ io = [
 test = [
     "aiohttp",
     "distributed",
+    "pandas",
```

Comment:
should this be added to complete as well?

Reply:
I would suggest there are too many potential awkward workflows that do not need pandas for it to be even a suggested dep. If you know you will be needing pandas, awkward-pandas will eventually be the way to go.

Reply:
Yea I think we can treat it how we treat s3fs, for example. If you call `dak.from_json("s3://...")` without `s3fs` installed then there's an error message instructing you to install it.

Reply:
I think I was confused here. See https://github.com/dask-contrib/dask-awkward/issues/212